engine/src/main/scala/cromwell/engine/backend/jes/JesBackend.scala

```diff
@@ -555,7 +563,10 @@ case class JesBackend(actorSystem: ActorSystem)
          |echo $$? > $rcPath
        """.stripMargin.trim
 
-    def attemptToUploadObject(priorAttempt: Option[Unit]) = authenticateAsUser(backendCall.workflowDescriptor) { _.uploadObject(backendCall.gcsExecPath, fileContent) }
+    def attemptToUploadObject(priorAttempt: Option[Unit]) = authenticateAsUser(backendCall.workflowDescriptor) { implicit gcsFs =>
+      backendCall.gcsExecPath.write(fileContent)
+      ()
```

Comment:
This is weird. Why not set your method return type explicitly instead of `return Unit`.